tools/ui-components/package.json

```diff
@@ -27,6 +27,9 @@
   "devDependencies": {
     "@babel/core": "7.21.4",
     "@babel/preset-env": "7.21.4",
+    "@fortawesome/free-solid-svg-icons": "6.4.0",
+    "@fortawesome/react-fontawesome": "0.2.0",
+    "@headlessui/react": "1.7.13",
```

Comment:
These should not be devDependencies.

Reply:
Yeab, can we move `@babel/preset-typescript` to dev dependency too?

it's used for config, we don't need to add it as dependecy https://babeljs.io/docs/babel-preset-typescript

Reply:
Yes that would be great.

Reply:
done 👍